backend/auth/tokenAuth.go

```diff
@@ -88,6 +92,23 @@ func VerifyNotLoggedIn(next http.Handler) http.Handler {
 	})
 }
 
+func InitialAdminCheck(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		var count int64
+		db.DB.Model(&models.ActiveTokens{Id: uuid.Max}).Count(&count)
```

Comment:
Is using uuid.Max a good approach? Why can't we just read in the environment variables like we did in the setup?

Reply:
We could generate the initial admin's token based on a UUID passed in an environment variable, but I don't think there is any benefit. It would be more inconvenient for a user to set up and it would not improve security, since you get redirected to the correct token registration page anyway and this flow only happens when there is no user data in the system.

Reply:
Im just saying why cant we use a random UUID?

Reply:
why do we care what the uuid is, there are no users, that being said I feel like we could totally just make it random no?

Reply:
actually on second thought, then u need to actually grab the uuid if its random so its probably more hassle for zero

Reply:
he selects by uuid later so he just needs to know if i guess

Reply:
yea but why tho, feels like extra work we don't need to do

Reply:
cuz if he sets it to a random token, he'd need to store what that random token is, ig in memory or smthn, in order for the select to check if that specific token is made.

Reply:
> actually on second thought, then u need to actually grab the uuid if its random so its probably more hassle for zero benefit

Yeah, that's the idea. The benefit of a random UUID is security through its randomness, since a bad actor would need to guess a correct UUID to steal a user's registration. In the initial admin flow however, we're redirecting any user directly to the registration page so the value of the UUID does not matter. For this flow, the randomness has no benefit.

By using a fixed UUID we gain two benefits:
1. We make it more explicit that this "special-case" registration token has no expiration.
2. We can more easily redirect the user to the initial admin registration page without needing to find the correct UUID.